programs/drift/src/state/user.rs

```diff
@@ -1716,7 +1716,9 @@ pub struct UserStats {
     /// Second bit: 1 if user was referred, 0 otherwise
     pub referrer_status: u8,
     pub disable_update_perp_bid_ask_twap: bool,
-    pub padding1: [u8; 2],
+    pub padding1: [u8; 1],
+    /// whether the user has a FuelSweep account
+    pub fuel_sweep_exists: bool,
```

Comment:
nit: maybe make this u8 in case we want to do bit flag in future (more optionality)